src/autoscaler/db/sqldb/sqldb_suite_test.go

```diff
@@ -378,3 +383,18 @@ func validateLockNotInDB(owner string) error {
 	}
 	return fmt.Errorf("lock exists with owner (%s)", owner)
 }
+
+func formatPolicyString(policyStr string) string {
+	scalingPolicy := &models.ScalingPolicy{}
+	err := json.Unmarshal([]byte(policyStr),&scalingPolicy)
+	if err != nil {
+		fmt.Errorf("failed to unmarshal policyJson string %s", policyStr)
+		return ""
+	}
+	policyJsonStr, err := json.Marshal(scalingPolicy)
+	if err != nil {
+		fmt.Errorf("failed to marshal ScalingPolicy %v", scalingPolicy)
+		return ""
+	}
+	return string(policyJsonStr)
```

Comment:
what is this for?

Reply:
Since the json data in mysql and postgres is different, for example, in mysql, it will remove blanks in the json object but in postgres it will keep the blanks.  The function is used to parse the policy json to the same format.